src/main/java/uk/gov/hmcts/reform/idam/web/model/SelfRegisterRequest.java

```diff
@@ -10,10 +11,23 @@
 @EqualsAndHashCode
 public class SelfRegisterRequest {
 
+    /**
+     * The blacklisted characters from first name and last name fields.
+     * If you modify any of these, please remember to update the equivalent messages.
+     */
+    private static final String BLACKLISTED_CHARACTERS = "\\*\\(\\)!/;:@#£\\$%=\\+";
+
+    /**
+     * Matches empty strings or strings longer than 1 character that don't include blacklisted characters nor digits.
```

Comment:
Does it need to match empty strings if you're also using @NotEmpty?

Reply:
It matches for empty string so it passes validation on @Pattern while failing validation on @NotEmpty rather than failing on both. 
This results in just one error message shown to the user and is there to keep consistency with the email box validation behaviour.